Userland/Applets/ClipboardHistory/ClipboardHistoryModel.h

```diff
@@ -41,6 +44,11 @@ class ClipboardHistoryModel final : public GUI::Model
     void clear();
     bool is_empty() { return m_history_items.is_empty(); }
 
+    ErrorOr<void> read_from_file(DeprecatedString const& path);
+    ErrorOr<void> write_to_file(bool rewrite_all);
+
+    ErrorOr<void> invalidate(bool rewrite_all);
```

Comment:
This line causes a compilation error when compiling with Clang, which is why the CI is failing.
```
.../Userland/Applets/ClipboardHistory/ClipboardHistoryModel.h:50:19: error: 'ClipboardHistoryModel::invalidate' hides overloaded virtual function [-Werror,-Woverloaded-virtual]
    ErrorOr<void> invalidate(bool rewrite_all);
```

Reply:
I know, I just didn't have time to correct it. Plus, it's surprising, as I'm pretty sure that we do that in other places.